backend/src/Controllers/controllerUtils.ts

```diff
@@ -0,0 +1,39 @@
+import { IRequest, reqIdSuffix } from "../IRequest"
+import { Unauthorized } from "@curveball/http-errors/dist"
+import { Election, electionValidation } from "../../../domain_model/Election";
+import Logger from "../Services/Logging/Logger"
+import { BadRequest, InternalServerError } from "@curveball/http-errors";
+import { Request, Response } from 'express';
+
+var jwt = require('jsonwebtoken')
+
+export function expectUserFromRequest(req:IRequest ):any {
```

Comment:
This should be a few functions I think. One to get user, one to expect user, and one to get user_auth. auth.controllers.ts has getUser which gets the user and user_auth, and has isLoggedIn. 

getUser (minus the part to get user_auth) could probably just be middleware that we add to the main app on all requests.